src/lib/handlers/db.js

```diff
@@ -1,5 +1,11 @@
 import { db } from '$lib/firebase/firebase';
-import { collection, doc, getDoc, getDocs, setDoc } from 'firebase/firestore';
+import { collection, doc, getDoc, getDocs, setDoc, deleteDoc } from 'firebase/firestore';
+import { authStore, setLoading } from '../store/store';
+
+let user;
+authStore.subscribe((curr) => {
+	user = curr?.user;
+});
 
```

Comment:
This is fine for now, but we will probably need to think about abstracting getter methods from store

Reply:
Makes sense. Thanks for pointing it out.